st2actions/st2actions/notifier/notifier.py

```diff
@@ -112,12 +111,12 @@ def _post_notify_triggers(self, liveaction=None, execution_id=None):
                 liveaction=liveaction, execution_id=execution_id,
                 notify_subsection=notify.on_complete,
                 default_message_suffix='completed.')
-        if liveaction.status == LIVEACTION_STATUS_SUCCEEDED and notify.on_success:
+        if liveaction.status == action_constants.LIVEACTION_STATUS_SUCCEEDED and notify.on_success:
             self._post_notify_subsection_triggers(
                 liveaction=liveaction, execution_id=execution_id,
                 notify_subsection=notify.on_success,
                 default_message_suffix='succeeded.')
-        if liveaction.status == LIVEACTION_STATUS_FAILED and notify.on_failure:
+        if liveaction.status == action_constants.LIVEACTION_STATUS_FAILED and notify.on_failure:
```

Comment:
In my PR I also introduced `action_constants.COMPLETED_STATES` so we don't copy and paste and duplicate those constants all over...